packages/world/ts/config/v2/world.ts

```diff
@@ -0,0 +1,77 @@
+import { evaluate, narrow } from "@arktype/util";
+import {
+  UserTypes,
+  Enums,
+  StoreConfigInput,
+  resolveStoreConfig,
+  validateStoreTablesConfig,
+  resolveStoreTablesConfig,
+  extendedScope,
+} from "@latticexyz/store/config";
+import { get } from "@latticexyz/store/ts/config/v2/generics";
+import { AbiTypeScope } from "@latticexyz/store/ts/config/v2/scope";
+import { resolveTableConfig } from "@latticexyz/store/ts/config/v2/table";
+
+export type WorldConfigInput<userTypes extends UserTypes = UserTypes, enums extends Enums = Enums> = evaluate<
+  StoreConfigInput<userTypes, enums> & {
+    namespaces?: NamespacesInput;
+  }
+>;
+
+export type NamespacesInput = { [key: string]: NamespaceInput };
+
+export type NamespaceInput = Omit<StoreConfigInput, "userTypes" | "enums">;
```

Comment:
do you think this will be annoying to maintain vs the inverse? `Pick<StoreConfigInput, "tables">`?

Reply:
agree it might be nicer to explicitly pick the things we want there than adding a store config option and forgetting about it here